spec/models/snippet_spec.rb

```diff
@@ -25,8 +25,8 @@
     end
 
     it "has a user_id" do
-      user = FactoryGirl.build_stubbed(:user)
-      snippet = FactoryGirl.build_stubbed(:snippet) 
+      user = FactoryGirl.build_stubbed(:user, id: 2)
```

Comment:
I like it. I'm using shoulda matchers in this spec on line 5, so maybe this test is redundant?